src/test/java/com/microsoft/graph/functional/OneNoteTests.java

```diff
@@ -429,6 +434,59 @@ public void testMultipartPost(){
             fail("Unable to write to output stream");
         }
     }
+    
+    /**
+     * Test posting multipart content to a page
+     */
+    @Test
+    public void testMultipartHeadersMapPost(){
```

Comment:
rename to testMultipartPostWithHeadersMap

Reply:
Renamed unit test function to testMultipartPostWithHeadersMap.